src/test/java/org/apache/cassandra/sidecar/routes/ExtractHostAddressWithoutPortTest.java

```diff
@@ -30,15 +30,15 @@
     @Test
     void testAddressWithIPv4Host()
     {
-        final String host = AbstractHandler.extractHostAddressWithoutPort("127.0.0.1");
-        assertEquals("127.0.0.1", host);
+        final String host = AbstractHandler.extractHostAddressWithoutPort("localhost");
```

Comment:
revert changes in this file? this is actually changing the test as described by the name of the test

Reply:
Done